lib/datastore/datastore_container.es6.js

```diff
@@ -160,114 +287,6 @@ foam.CLASS({
   ],
 });
 
-foam.ENUM({
```

Comment:
Did this need to be moved? Could it have been a separate PR to make comparing old and new possible?

Reply:
I just wanted to keep it above its first use. Code should still work with it in its old place, though.

Reply:
Done.